sensor/common/message/message.go

```diff
@@ -0,0 +1,49 @@
+package message
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+)
+
+// ExpiringMessage is a wrapper on central.MsgFromSensor with the addition of a Context.
+// The context will be cancelled when the message is expired and should no longer be sent
+// to Central.
+type ExpiringMessage struct {
+	*central.MsgFromSensor
+	Context context.Context
+}
+
+// New creates an ExpiringMessage with msg and context.Background.
+func New(msg *central.MsgFromSensor) *ExpiringMessage {
```

Comment:
Maybe 
```suggestion
func NewExpiring(msg *central.MsgFromSensor) *ExpiringMessage {
```
??
Not sure about the right naming here, but I was slightly confused seeing `message.New()` in the wild.

Reply:
In this case this is a non expiring message. I'm not sure if `NewNonExpiring` or some antonym of `Expiring` like `NewLastingMessage` would be beneficial. 

The way I see it, `message.New` creates a component message. Which is straight forward. You don't need to worry about what are the properties of this message, they are all defaulted. Perhaps the crux of the confusion is in the name of the struct?

Reply:
Yes, I think there is a kind of pattern when building constructors that we prepend `New` to the name of the struct. I see no issue in calling a message that has expiration in `+inf` a `NewExpiring` message.